Student/Student/Courses/Syllabus/SyllabusActionableItemsPresenter.swift

```diff
@@ -53,7 +53,7 @@ class SyllabusActionableItemsPresenter {
         }
     }()
 
-    init(env: AppEnvironment = .shared, view: SyllabusActionableItemsViewProtocol, courseID: String, sort: GetAssignments.Sort = .position) {
+    init(env: AppEnvironment = .shared, view: SyllabusActionableItemsViewProtocol, courseID: String, sort: GetAssignments.Sort = .dueAt) {
```

Comment:
I changed this and below because it was defaulting to position which is not what we want on the syllabus.